### ISSUE TITLE
About  manual annotations 

### ISSUE THREAD

**Issue by chenhaomingbob** [facebookresearch/PoseWarper#28]
Thanks for your wonderful work.

In Table1 of your paper, we find that you use the manual annotations.
I can't find these annotations file, could you provide it for me?

**Comment by gberta** [comment]
I believe that we used "posetrack_train_N5_per_video.json", which is provided in the zip file, which is included in this repo. However, please double check to make sure. I'm not 100% sure since it's been a long time since I worked on this project. I would urge you to look into the training scripts, and configuration files associated with the experiments that we run to generate Table 1.

**Comment by chenhaomingbob** [comment]
Thanks, I find `posetrack_train_N5_per_video.json` in folder `posetrack17_json_files`. 

But when I run the experiment `video pose propagation`, meting some confusion。

#### I use the following command ( inference video pose propagation by delta -3) :
`python tools\test.py --cfg
../experiments/posetrack/hrnet/w48_384x288_adam_lr1e-4_PoseWarper_inference_video_pose_propagation.yaml
OUTPUT_DIR
/media/jion/D/chm/Code/PoseWarper_supp_files/posetrack17_experiments/PoseWarper/V250_N5/out/
LOG_DIR
/media/jion/D/chm/Code/PoseWarper_supp_files/posetrack17_experiments/PoseWarper/V250_N5/log/
DATASET.NUM_LABELED_VIDEOS
250
DATASET.NUM_LABELED_FRAMES_PER_VIDEO
5
DATASET.JSON_DIR
/media/jion/D/chm/Code/PoseWarper_supp_files/posetrack17_json_files/
DATASET.IMG_DIR
/media/jion/D/chm/DataSet/PoseTrack2017/images_renamed/
TEST.MODEL_FILE
/media/jion/D/chm/Code/PoseWarper_supp_files/posetrack17_experiments/PoseWarper/V250_N5/out/posetrack/pose_hrnet/w48_384x288_adam_lr1e-4_PoseWarper_train/final_state.pth
POSETRACK_ANNOT_DIR
/media/jion/D/chm/Code/PoseWarper_supp_files/posetrack17_annotation_dirs/val_withJson_consecutive/val/
MODEL.EVALUATE
False
MODEL.TIMESTEP_DELTA
-3
SAVE_PREDS
True
PRINT_FREQ
5000`

#### My Confusion
This situation is that model concat_input contain image `ref_x`, image `sup_x`, and ground truth(gt) heatmaps  `gt_x`  correspond to image `ref_x`.  After seed image `ref_x` and image `sup_x` through HRNet, these become heatmaps `ref_x`  and heatmaps `sup_x`. **And in** 
https://github.com/facebookresearch/PoseWarper/blob/43b1aa373d945abaf9e9a1f70953cc7411796a1a/lib/models/pose_hrnet.py#L636
**replace `sup_x` by `gt_x`， this operate make me confusion**. In my view, this operation means that use the difference between Frame A and Frame B to warp  **Frame A gt heatmaps**, and finally we generate Frame A heatmaps. Why not use the difference between Frame A and Frame B to warp  **Frame B gt heatmaps**. Specially, use `gt_ref_x` correspond to image `ref_x` gt heatmaps replace `sup_x`. 

Maybe I understanding wrong, I hope you can solve my doubts.

**Comment by chenhaomingbob** [comment]
Thanks, I found it.
